abi_wrapper.go

```diff
@@ -0,0 +1,37 @@
+package web3
+
+import (
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gochain-io/gochain/v3/accounts/abi"
+)
+
+func getReader(contractFile string) io.Reader {
+	BundledContracts := map[string]string{
```

Comment:
Can this be a global unexported var?
Using backtick strings removes the need for escape characters and retains the formatting.